packages/kit/src/runtime/server/page/actions.js

```diff
@@ -142,7 +141,7 @@ export async function handle_action_request(event, server) {
 		});
 		return {
 			type: 'error',
-			error: new HttpError(405, 'POST method not allowed. No actions exist for this page')
+			error: new SvelteKitError(405, 'POST method not allowed. No actions exist for this page')
```

Comment:
we should probably use the stand HTTP error codes consistently

```suggestion
			error: new SvelteKitError(405, 'Method Not Allowed', 'POST method not allowed. No actions exist for this page')
```